docs/apis/avalanchego/public-api-server.md

```diff
@@ -47,7 +47,18 @@ avalanche.setRequestConfig("withCredentials", true);
 
 ## Availability
 
-Usage of public API nodes is free and available to everyone without any authentication or authorization. Rate limiting is present, but many of the API calls are cached, and the rate limits are quite high. If your application is running up against the limits, please [contact us](https://chat.avalabs.org).
+Usage of public API nodes is free and available to everyone without any authentication or authorization. Rate limiting is present, but many of the API calls are cached, and the rate limits are quite high. If your application is running up against the limits, please [contact us](https://chat.avalabs.org) or try using a 3rd party RPC provider. 
+
+## 3rd Party Public RPC Providers
```

Comment:
Please move this section to the bottom of this document.

Reply:
Done in latest commit.